packages/njsscan/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-njsscan
+_pkgname=njsscan
+pkgver=0.2.0
+pkgrel=1
+pkgdesc='A static application testing (SAST) tool that can find insecure code patterns in your node.js applications.'
+arch=('any')
+url='https://pypi.org/project/jschema-to-python/#files'
```

Comment:
Wrong URL ? Is it https://pypi.org/project/njsscan/#files ?

Reply:
Yes my bad. Adjusted. Have not checked in yet.